js/ui/selection/selection.strategy.js

```diff
@@ -69,8 +69,28 @@ export default Class.inherit({
         return this.selectedItemKeys(keys, preserve, isDeselect, isSelectAll);
     },
 
+    _removeTemplateProp: function(filterItem) {
+        if(isObject(filterItem) && Object.prototype.hasOwnProperty.call(filterItem, 'template')) {
```

Comment:
do we really need to check if `template` property exists in the item? Think, we can always delete it